pontoon/pretranslation/pretranslate.py

```diff
@@ -27,6 +27,16 @@ def __init__(self, locale):
         self.services = []
         self.locale = locale
 
+    def visit_Attribute(self, node):
+        is_accesskey = (
+            node.id.name.endswith("accesskey")
+            and isinstance(node.value.elements[0], ast.TextElement)
+            and len(node.value.elements[0].value) <= 1
```

Comment:
This part of the check is taken from the frontend:
https://github.com/mathjazz/pontoon/blob/2717-accesskey-opt-out/translate/src/modules/translationform/components/TranslationForm.tsx#L49